src/simulator/SimulatorModules.tsx

```diff
@@ -155,6 +155,13 @@ const CollapsibleModule = ({
   const disclosure = useDisclosure();
   const intl = useIntl();
   const [, setRouterState] = useRouterState();
+  const setPanelFocus = () => {
+    const sidebarElement = document.querySelector("[aria-label='Sidebar']");
```

Comment:
Querying globally like this is fragile and presumably breaks if you change language.

I think this should be part of the navigation state.

Reply:
Added `focusTab` to navigation state here -> [`f993ac8`](https://github.com/microbit-foundation/python-editor-next/pull/898/commits/f993ac8d7f1e320517d1ad4a94e757f16f97dfff).